src/adapters/notification_adapter.ts

```diff
@@ -233,6 +239,102 @@ export class NotificationAdapter {
     return this.createSubscription(eventName, sanitationCallback, subscribeOnce);
   }
 
+  public onCallActivityWaiting(
+    identity: IIdentity,
+    callback: Messages.CallbackTypes.OnCallActivityWaitingCallback,
+    subscribeOnce: boolean,
+  ): Subscription {
+
+    const eventName: string = Messages.EventAggregatorSettings.messagePaths.callActivityReached;
```

Comment:
Please make use of type-inference.

```suggestion
    const eventName = Messages.EventAggregatorSettings.messagePaths.callActivityReached;
```